enterprise/frontend/src/embedding-sdk/components/public/CreateDashboardModal/CreateDashboardModal.tsx

```diff
@@ -49,6 +49,6 @@ const CreateDashboardModalCoreWithLoading = _.compose(
   }),
 )(CreateDashboardModalCore);
 
-export const CreateDashboardModal = _.compose(withPublicComponentWrapper)(
+export const CreateDashboardModal = withPublicComponentWrapper(
```

Comment:
Previously `compose` doesn't have any type 
![image](https://github.com/user-attachments/assets/b4d450a0-c9b6-40e9-97c8-f8e11d07cd58)

With this PR, we'll get a type, but it's still wrong, because it seems it doesn't support types with generics.
![image](https://github.com/user-attachments/assets/9f0b7c2c-683e-47a5-ada1-b5c3c2a44e34)

So, I ditch `compose` since we only have 1 function here anyway.